src/Transport/FallbackSnmpClient.php

```diff
@@ -55,10 +55,20 @@ static function (SnmpClient $client) use ($oid, $maxRepetitions) : array {
         );
     }
 
+    /** @inheritDoc */
+    public function batch(Request ...$requests) : array
+    {
+        return $this->tryClients(
+            static function (SnmpClient $client) use ($requests) : array {
+                return $client->batch(...$requests);
+            }
+        );
+    }
+
     /**
-     * @param callable(SnmpClient): array<string, mixed> $requestCallback
+     * @param callable(SnmpClient): array<mixed> $requestCallback
```

Comment:
no array key?

Reply:
mělo by tam být `array<string, mixed>|list<array<string, mixed>>`, ale s tím si prostě phpstan nedokázal poradit

Reply:
dal jsi to do zavorek?

Reply:
yes, problém nebyl se syntaxí, ale že nechápal ty typy okolo ... no klidně zkus u sebe, jestli přijdeš na to jak to fixnout tak ideal...

Reply:
ten commit co jsi přidal nedává smysl, viz ten typ co jsem psal nahoře ...tvůj je špatně

Reply:
ja jsem nenasel, ze by to nekde vracelo `array<string, mixed>`

Reply:
get, getNext a walk to vrací

Reply:
ok mi nejak nefunguje ide lol